src/overview.md

```diff
@@ -42,16 +42,15 @@ allowing the use of a `dynamic` type combined with runtime checks,
 which can be useful during experimentation or
 for code that needs to be especially dynamic.
 
-Dart offers [sound null safety](/null-safety),
-meaning that values can’t be null unless you say they can be.
+Dart offers [sound null safety](/null-safety).
+This means values can’t be null unless you say they can be.
 With sound null safety, Dart can protect you from
 null exceptions at runtime through static code analysis.
 Unlike many other null-safe languages,
-when Dart determines that a variable is non-nullable,
-that variable is _always_ non-nullable.
+when Dart determines that a variable can't be null,
+that variable can never be null.
 If you inspect your running code in the debugger,
-you’ll see that non-nullability is retained at runtime
-(hence _sound_ null safety).
+you see that runtime retains this non-nullability; hence _sound_ null safety.
```

Comment:
It doesn't make sense to say that "runtime retains". You could technically say "the runtime retains" but developers shouldn't have to worry about "the runtime".

```suggestion
you see that non-nullability is retained at runtime; hence _sound_ null safety.
```